src/drivers/uavcan/actuators/esc.cpp

```diff
@@ -230,3 +230,19 @@ void UavcanEscController::orb_pub_timer_cb(const uavcan::TimerEvent &)
 		_esc_status_pub = orb_advertise(ORB_ID(esc_status), &_esc_status);
 	}
 }
+
+uint8_t UavcanEscController::check_escs_status()
+{
+	int esc_status_flags = 0;
+
+
+	for (int index = 0; index < esc_status_s::CONNECTED_ESC_MAX; index++) {
+
+		if (_esc_status.esc[index].timestamp > 0 && hrt_elapsed_time(&_esc_status.esc[index].timestamp) < 800000.0f) {
```

Comment:
2 more things you can do here: pull the `hrt_absolute_time` call out of the loop: `hrt_abstime now = hrt_absolute_time();`, and use the time literals: `now - _esc_status.esc[index].timestamp < 800_ms) {`
(add `using namespace time_literals;` at the top of the file).